packages/node_modules/@webex/plugin-meetings/src/stats/transformer.js

```diff
@@ -10,17 +10,19 @@ StatsTransformer.simplify = (stat) => find(keys(DEFAULT_TRANSFORM_REGEX), (key)
   let value = (DEFAULT_TRANSFORM_REGEX[key].regex.test(stat.id) ? key : false);
 
   if (value) {
-    // some types need to check against a type field as well
-    if (DEFAULT_TRANSFORM_REGEX[value].profiler) {
-      const mapStatToReadableType = stat[DEFAULT_TRANSFORM_REGEX[value].profiler.type];
+    // TODO: wait a year to see if webrtc readds their different implementation for rtp/rtc/transport stats
```

Comment:
Is there a link we could add to this comment to track the work being done on this, or the limitation?